activity/rest/activity.go

```diff
@@ -119,7 +120,22 @@ func (a *RESTActivity) Eval(context activity.Context) (done bool, err error) {
 		req.Header.Set("Content-Type", contentType)
 	}
 
-	client := &http.Client{}
+	// Set the proxy server to use, if supplied
+	proxy := context.GetInput(ivProxy)
+	var client *http.Client
+	var proxyValue, ok = proxy.(string)
```

Comment:
the ok pattern will take care of the nil, this should suffice, also for empty strings len() is used:
 proxyValue, ok := proxy.(string)
if ok && len(proxyValue) > 0{
...
} else {
...
}